src/profiler.cpp

```diff
@@ -1214,6 +1214,10 @@ Error Profiler::start(Arguments& args, bool reset) {
     }
 
     if (_event_mask & EM_ALLOC) {
+        if (args._all && VM::hotspot_version() < 11) {
+            Log::debug("'live' option was quietly removed, only supported on OpenJDK 11+");
```

Comment:
This is not accurate: `_live` is also supported with OpenJ9.
Why not checking `_all` in `AllocTracer::check`?